index.js

```diff
@@ -30,6 +37,60 @@ app.get('/ideas', (req, res) => {
     res.status(200).json(getIdeas());
 });
 
+app.get('/ideas?sortBy=totalDonationSum&sortOrder=desc', (req, res) => {
```

Comment:
This is incorrect.

`/ideas`  is the path.

`?sortBy=totalDonationSum&sortOrder=desc` are query parameters.

Try to figure out how to access query parameters in Express.js.

Reply:
Are we talking about query parser (query string parser)?